### ISSUE TITLE
Портит ссылки, в которых есть не латинские символы

### ISSUE THREAD

**Issue by nokeya** [nokeya/direct-links-out#9]
По крайней мере в vk

**Comment by IRainman** [comment]
Да, тоже столкнулся с этой проблемой. Пока отключил скрипт. Жду ответа от разработчика ,возможно это серьёзный баг. Кстати, поддержку ссылок с не латинскими доменами (punycode) тоже надо проверить ибо сейчас их уже много и плохо если с ними проблемы.

**Comment by safarumebe** [comment]
nokeya, а можете добавить свой емейл куда скинуть у меня куча предложении и ещё своих редиректов!
Хотелось бы ещё флаги вкл/выкл редиректа.

Или напишите мне
safarumebe [at] protonmail.com

**Comment by IRainman** [comment]
Ждём хоть какого то прогресса в этой области. Автор, вы вообще тут или проект умер?

**Comment by nokeya** [comment]
@safarumebe  
Лучше все-таки пишите issue сюда на гитхаб
@IRainman 
Не умер, но все таки полгода точно не развивался - не было времени и желания. Теперь надеюсь опять пойдет разработка

**Comment by atorn1** [comment]
unescape --> decodeURIComponent
